src/common/AndroidUtils.ts

```diff
@@ -173,51 +172,6 @@ export class AndroidUtils {
         return Promise.resolve(PlatformConfig.androidConfig().supportedDeviceTypes);
     }
 
-    /**
-     * Attempts to fetch all available Android virtual devices.
-     *
-     * @returns An array of all available Android virtual devices.
-     */
-    public static async fetchEmulators(logger?: Logger): Promise<AndroidVirtualDevice[]> {
-        let devices: AndroidVirtualDevice[] = [];
```

Comment:
These are not needed anymore since they've been refactored into `AndroidDevice` and `AndroidDeviceManager`